server/controllers/finance/debtorGroups.js

```diff
@@ -111,20 +111,26 @@ function getDetail(req, res, next) {
 
 /**
 * GET /debtor_groups/ => all debtor groups
-* GET /debtor_groups/?locked={1|0} => only locked or not locked debtor groups
+* GET /debtor_groups/?locked={1|0}&is_convention={1|0} => locked or convention criteria
```

Comment:
As these query strings get more options, it might be easier to lay comments out in this fashion:
```js
/**
 * GET /debtor_groups
 * @param is_convention  {boolean} (0 | 1) filter debtor groups in the convention column
 * @param locked {boolean} (0 | 1) filters locked debtor groups
 */
```